src/mumble/AudioInput.cpp

```diff
@@ -1102,12 +1104,24 @@ void AudioInput::encodeAudioFrame(AudioChunk chunk) {
 			p->setTalking(Settings::Shouting);
 	}
 
-	if (Global::get().s.bTxAudioCue && Global::get().uiSession != 0) {
+	if (Global::get().uiSession != 0 && (Global::get().s.bTxAudioCue || Global::get().s.bTxMuteCue)) {
 		AudioOutputPtr ao = Global::get().ao;
-		if (bIsSpeech && !bPreviousVoice && ao)
-			ao->playSample(Global::get().s.qsTxAudioCueOn);
-		else if (ao && !bIsSpeech && bPreviousVoice)
-			ao->playSample(Global::get().s.qsTxAudioCueOff);
+
+		if (ao) {
+			if (Global::get().s.bTxAudioCue) {
+				if (bIsSpeech && !bPreviousVoice)
+					ao->playSample(Global::get().s.qsTxAudioCueOn);
+				else if (!bIsSpeech && bPreviousVoice)
+					ao->playSample(Global::get().s.qsTxAudioCueOff);
```

Comment:
```suggestion
				if (bIsSpeech && !bPreviousVoice) {
					ao->playSample(Global::get().s.qsTxAudioCueOn);
				} else if (!bIsSpeech && bPreviousVoice) {
					ao->playSample(Global::get().s.qsTxAudioCueOff);
				}
```
When we're changing these already, we might as well add these curly brackets (we prefer always having them)

Reply:
Done.